exchanges/exchanges_test.go

```diff
@@ -231,6 +232,31 @@ func TestPoloniexWebsocket(t *testing.T) {
 	}
 }
 
+type tDoer struct {
+	responses []*http.Response
+}
+
+func (d *tDoer) Do(*http.Request) (*http.Response, error) {
+	if len(d.responses) == 0 {
+		return nil, fmt.Errorf("no test response queued")
+	}
+	resp := d.responses[0]
+	d.responses = d.responses[1:]
+	return resp, nil
+}
+
+func (d *tDoer) queue(body interface{}) *http.Response {
+	b, err := json.Marshal(body)
+	if err != nil {
+		panic("tDoer.queue error:" + err.Error())
+	}
+	resp := &http.Response{
+		Body: io.NopCloser(bytes.NewReader(b)),
+	}
+	d.responses = append(d.responses, resp)
+	return resp
+}
+
 type testBittrexConnection struct {
 	xc           *BittrexExchange
 	subscription hubs.ClientMsg
```

Comment:
current linters on my machine have started fussing about the `conn.subscription = subscription` in Send being ineffective/unused, and it really comes back to this field being unused.  Can we just remove this?